scripts/renameJStoJSX.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+ echo "Fetching misnamed files"
+FILES_TO_RENAME=$(grep --exclude-dir={node_modules,scripts} --exclude=\*.{jsx,spec.js} --exclude={index.js,routes.js} -Rl 'import React' ./)
```

Comment:
```suggestion
FILES_TO_RENAME=$(grep --exclude-dir={node_modules,scripts} --exclude=\*.{jsx} --exclude={index.js,routes.js} -Rl 'import React' ./)
```
La règle eslint dit aussi qu'il faut mettre les tests en jsx s'ils contiennent du jsx.